### PR TITLE
deploy.ps1: allow VM SKU sizes to be passed as parameter

### PR DESCRIPTION
**Pull request by saramonteiro**
# Description

This PR is a workaround for the issue https://github.com/Azure/Industrial-IoT/issues/1203. 
It only adds the feature of passing the VM SKU for the gateway host and OPC PLC server simulator as parameters.
A complete(and better) fix would include filter out the non Generation 1 SKU from the VM SKU list.

# Test

I got a successful deploy by using the following command:

`.\deploy -gatewayVMSKU Standard_D1_v2 -OPCPLCVMSKU Standard_D2_v2`

# Note:

If you choose to pass the SKU as parameters, make sure they're available on your chosen region by running the following snippet on your Power Shell:

``` bash
            Get-AzVMSize -Location "East US" `
            | Where-Object {
                ($_.NumberOfCores -ge 1) -and `
                ($_.MemoryInMB -ge 2048) -and `
                ($_.OSDiskSizeInMB -ge 1047552) -and `
                ($_.ResourceDiskSizeInMB -ge 4096)
            } `
```

### CODE REVIEW

File: deploy/scripts/deploy.ps1 [hunk 1]
```diff
@@ -70,6 +70,16 @@
 
  .PARAMETER numberOfWindowsGateways
     Number of Windows gateways to deploy into the simulation.
+
+ .PARAMETER gatewayVMSKU
```
Comment (karok2m) [inline deploy/scripts/deploy.ps1]: @saramonteiro, let's make name of the variable came-case to align with the rest. Something like `gatewayVmSku`.

File: deploy/scripts/deploy.ps1 [hunk 2]
```diff
@@ -70,6 +70,16 @@
...
+    Virtual machine SKU size that hosts simulated edge gateway.
+    Suggestion: use VM with at least 2 cores and 8 GB of memory.
+    Must Support Generation 1.
+
+ .PARAMETER OPCPLCVMSKU
```
Comment (karok2m) [inline deploy/scripts/deploy.ps1]: @saramonteiro, let's make name of the variable came-case to align with the rest. Something like `opcPlcVmSku`.

Comment (saramonteiro) [reply deploy/scripts/deploy.ps1]: Fixed, when it's ok, let me know that I squash my commits.

**Review by karok2m — APPROVED** [review]

**Review by cristipogacean — APPROVED** [review]

**Comment by karok2m** [comment]
Here is the successful run of [Azure.Industrial-IoT pipeline](https://msazure.visualstudio.com/One/_build/results?buildId=51048751&view=results) for this PR.

**Pull request merged**